components/popover/index.js

```diff
@@ -96,23 +96,39 @@ class Popover extends Component {
 	}
 
 	focus() {
-		const { focusOnMount = true } = this.props;
-		if ( ! focusOnMount ) {
-			return;
+		const { focusOnMount } = this.props;
+
+		// Boolean values for focusOnMount deprecated in 3.2.
+		if ( focusOnMount === true ) {
+			window.console.warn( '<Popover> component: focusOnMount should be false or a string as of Gutenberg 3.2. See: https://github.com/WordPress/gutenberg/tree/master/components/popover/README.md' );
 		}
 
-		if ( ! this.contentNode.current ) {
+		if ( ! focusOnMount || ! this.contentNode.current ) {
 			return;
 		}
 
 		// Without the setTimeout, the dom node is not being focused
 		// Related https://stackoverflow.com/questions/35522220/react-ref-with-focus-doesnt-work-without-settimeout-my-example
 		const focusNode = ( domNode ) => setTimeout( () => domNode.focus() );
 
-		// Find first tabbable node within content and shift focus, falling
-		// back to the popover panel itself.
-		const firstTabbable = focus.tabbable.find( this.contentNode.current )[ 0 ];
-		focusNode( firstTabbable ? firstTabbable : this.contentNode.current );
+		if ( focusOnMount === 'firstElement' || focusOnMount === true ) {
```

Comment:
If we do the above then let's make a note here that this `focusOnMount === true` check can be removed in 3.4.